File: okj/rendszeruzemelteto/berek2020/c#/Dolgozo.cs
public class Dolgozo {

    public readonly string nev;
    public readonly string nem;
    public readonly string munkaReszleg;
    public readonly int munkabaLepesEv;
    public readonly int munkaBer;

    public Dolgozo(string line) {
        var split = line.Split(';');

        nev = split[0];
        nem = split[1];
        munkaReszleg = split[2];
        munkabaLepesEv = int.Parse(split[3]);
        munkaBer = int.Parse(split[4]);
    }
}

File: okj/rendszeruzemelteto/berek2020/java/Dolgozo.java
public class Dolgozo {

    public final String nev;
    public final String nem;
    public final String munkaReszleg;
    public final int munkabaLepesEv;
    public final int munkaBer;

    public Dolgozo(String line) {
        var split = line.split(";");

        nev = split[0];
        nem = split[1];
        munkaReszleg = split[2];
        munkabaLepesEv = Integer.parseInt(split[3]);
        munkaBer = Integer.parseInt(split[4]);
    }
}